bld/CLMBuildNamelist.pm

```diff
@@ -4073,7 +4073,16 @@ sub setup_logic_fire_emis {
         if ( &value_is_true( $nl_flags->{'use_fates'} ) ) {
           $log->warning("Fire emission option $var can NOT be on when FATES is also on.\n" .
                       "  DON'T use the '--fire_emis' option when '--bgc fates' is activated");
-       }
+        } elsif ( ! &value_is_true( $nl_flags->{'use_cn'} ) ) {
+          $log->fatal_error("Fire emission option $var can NOT be on BGC SP Satellite Phenology.\n" .
+                           "  DON'T use the '--fire_emis' option when '--bgc sp' is activated");
```

Comment:
Is this doing the right thing here? Is this for both fire_emiss on and off?

Reply:
On review this looks correct.

But, I see a slight tweak of wording that would be useful.